src/bot/precomileInclude.jl

```diff
@@ -0,0 +1,120 @@
+
+"""
+    precompilePather(packageName::String)
+
+To get the path of precompile_packageName.jl file
+
+Written exclusively for SnoopCompile Github actions.
+# Examples
+```julia
+precompilePath, precompileFolder = precompilePather("MatLang")
+```
+"""
+function precompilePather(packageName::String)
+    return "\"../deps/SnoopCompile/precompile/precompile_$packageName.jl\"",
```

Comment:
`parcel` includes the package name as part of the file name, but when I copy the files to a project's `src/` directory I often strip the package name off and just call it `precompile.jl`. Is it possible to detect this case too?

Reply:
No. Now the precompile signatures should be kept in this particular path, otherwise the bot will not work.